db/seedfiles/or_ivl_life_events_seed.rb

```diff
@@ -0,0 +1,609 @@
+  title: "New offer of an HRA during the plan year"
+).tap do |qlek|
+  qlek.update_attributes(
+  tool_tip: "Someone has a new offer of a Health Reimbursement Arrangement starting during the plan year (for example, when newly hired)",
+  event_kind_label: "Date provider confirmed pregnancy",
+  action_kind: "transition_member",
+  market_kind: "individual",
+  ordinal_position: 12,
+  reason: "hra_midyear",
+  edi_code: "",
+  effective_on_kinds: ["first_of_next_month"],
+  pre_event_sep_in_days: 0,
+  post_event_sep_in_days: 60,
+  is_self_attested: false,
+  is_visible: true,
+  date_options_available: false
+  )
+end
+
+# TODO: Says first_of_next_month plan selection need to figure that out
+QualifyingLifeEventKind.find_or_create_by(
+  title: "Exceptional circumstances"
+).tap do |qlek|
+  qlek.update_attributes(
+  tool_tip: "An exceptional circumstance prevented someone from enrolling during open enrollment or a special enrollment period",
+  event_kind_label: "Date of event",
+  action_kind: "transition_member",
+  market_kind: "individual",
+  ordinal_position: 27,
+  reason: "exceptional_circumstance",
+  edi_code: "",
+  effective_on_kinds: ["first_of_next_month"],
+  pre_event_sep_in_days: 0,
+  post_event_sep_in_days: 60,
+  is_self_attested: false,
+  is_visible: true,
+  date_options_available: true
+  )
+end
+
+# TODO: Says first_of_next_month plan selection need to figure that out
+QualifyingLifeEventKind.find_or_create_by(
+  title: "Plan error"
+).tap do |qlek|
+  qlek.update_attributes(
+  tool_tip: "A material error related to plan benefits, service area, or premium influenced someone's health plan selection",
+  event_kind_label: "Date error was discoovered",
+  action_kind: "transition_member",
+  market_kind: "individual",
+  ordinal_position: 24,
+  reason: "enrollment_error_carrier",
+  edi_code: "",
+  effective_on_kinds: ["first_of_next_month"],
+  pre_event_sep_in_days: 0,
+  post_event_sep_in_days: 60,
+  is_self_attested: false,
+  is_visible: true,
+  date_options_available: true
+  )
+end
+
+# TODO: Says first of next month plan selection
+QualifyingLifeEventKind.find_or_create_by(
+  title: "Newly eligible or ineligible for APTC"
+).tap do |qlek|
+  qlek.update_attributes(
+  tool_tip: "A marketplace enrollee is found newly eligible or ineligible for advance premium tax credits, or someone who is not enrolled has a decrease in income that makes them eligible",
+  event_kind_label: "Date error was discoovered",
+  action_kind: "transition_member",
+  market_kind: "individual",
+  ordinal_position: 13,
+  reason: "income_change",
+  edi_code: "",
+  effective_on_kinds: ["first_of_next_month"],
+  pre_event_sep_in_days: 0,
+  post_event_sep_in_days: 60,
+  is_self_attested: false,
+  is_visible: true,
+  date_options_available: false
+  )
+end
+
+# TODO: Says first of next month plan selection
+QualifyingLifeEventKind.find_or_create_by(
+  title: "Denied MaineCare or Cub Care after enrollment period"
```

Comment:
very minor, but "MaineCare or Cub Care" should be "Oregon Health Plan" in this title